packages/site/src/componentList.ts

```diff
@@ -239,11 +254,13 @@ export const componentList = [
     title: "InputEmail",
     to: "/components/InputEmail",
     imageURL: "/InputEmail.png",
+    additionalMatches: ["Email Input"],
```

Comment:
I don't think it hurts to have these listed, but if someone searches "email", InputEmail will come up anyway

Reply:
Sounds good! I'll merge this and then I can take over + manage any conflict resolution on your branch 🫡